packages/stg-evm-usdt-v2/hardhat.config.js

```diff
@@ -0,0 +1,109 @@
+require('@nomicfoundation/hardhat-toolbox');
+require('@openzeppelin/hardhat-upgrades');
+require('dotenv').config();
+
+const accounts = {
+    mnemonic: process.env.MNEMONIC,
+};
+
+/** @type import('hardhat/config').HardhatUserConfig */
+module.exports = {
+    namedAccounts: {
+        deployer: {
+            default: 0,
+        },
+        alice: {
+            default: 1,
+        },
+        bob: {
+            default: 2,
+        },
+        carol: {
+            default: 3,
+        },
+    },
+    networks: {
+        celo: {
```

Comment:
For consistency purposes please name the networks the LayerZero way (`ethereum-mainnet` etc)

Reply:
will do!